src/cljs/main/broadfcui/page/library/library_page.cljs

```diff
@@ -17,6 +17,29 @@
    [broadfcui.utils :as utils]
    ))
 
+(defn- tcga-access-instructions []
+  [:span {}
+   [:p {} "For access to TCGA protected data please apply for access via dbGaP [instructions can be found "
+    [:a {:href "https://wiki.nci.nih.gov/display/TCGA/Application+Process"
+         :target "_blank"}
+     "here" icons/external-link-icon] "]."]])
+
+(defn- target-access-instructions []
+  [:span {}
+   [:p {} "For access to TARGET protected data please apply for access via dbGaP [instructions can be found "
+    [:a {:href "https://ocg.cancer.gov/programs/target/using-target-data"
+         :target "_blank"}
+     "here" icons/external-link-icon] "]."]])
+
+(defn- standard-access-instructions [data]
+  [:span {}
+   "Please contact "
+   [:a {:target "_blank"
+        :href (str "mailto:" (:library:contactEmail data))}
+    (:library:datasetCustodian data) " <" (:library:contactEmail data) ">"
+    (icons/icon {:style {:paddingLeft "0.25rem" :fontSize "80%"}} :email)]
+   " and request access for the "
+   (:namespace data) "/" (:name data) " workspace."])
```

Comment:
that would be ideal, but the endpoint is at the workspace level and requires that the workspace was shared w/ you, so it would have to be adjusted to work in lib. after going over this with Moran it did sound like all of this was just going to be a temporary solution until she is able to see it in action over time and refine it